src/datasets/load.py

```diff
@@ -1431,6 +1431,13 @@ def load_metric(
 
     Returns:
         `datasets.Metric`
+
+    Example:
+
+    ```py
+    >>> from datasets import load_metric
+    >>> accuracy = load_metric('accuracy')
```

Comment:
let's also show how it's used maybe ?
```suggestion
    >>> accuracy = load_metric('accuracy')
    >>> accuracy.compute(references=[1, 0], predictions=[1, 1])
    {'accuracy': 0.5}
```